app/lib/common/widgets/acter_icon_picker/acter_icon_widget.dart

```diff
@@ -26,28 +27,22 @@ class _ActerIconWidgetState extends State<ActerIconWidget> {
 
   @override
   Widget build(BuildContext context) {
-    if (widget.color != null) {
-      color.value = widget.color!;
-    }
-    if (widget.icon != null) {
-      icon.value = widget.icon!;
-    }
+    widget.color.let((p0) => color.value = p0);
+    widget.icon.let((p0) => icon.value = p0);
     return InkWell(
       borderRadius: BorderRadius.circular(100),
-      onTap: widget.onIconSelection == null
-          ? null
-          : () => showActerIconPicker(
-                context: context,
-                selectedColor: color.value,
-                selectedIcon: icon.value,
-                onIconSelection: (selectedColor, selectedIcon) {
-                  color.value = selectedColor;
-                  icon.value = selectedIcon;
-                  if (widget.onIconSelection != null) {
-                    widget.onIconSelection!(selectedColor, selectedIcon);
-                  }
-                },
-              ),
+      onTap: () => widget.onIconSelection.let(
+        (cb) => showActerIconPicker(
```

Comment:
same as above: if we don't have a callback we want there to be `null` not an empty function - as the latter behaves differently in inkwell.

```suggestion
      onTap: widget.onIconSelection.let(
        (cb) => () => showActerIconPicker(
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2233/commits/313bf8d1392ea01c0add35d03a523613e590136f